DEV_GUIDE.md

```diff
@@ -337,8 +337,8 @@ has been applied ineffectively.
 * `SKIP_TEARDOWN`: variable for development purposes to avoid keep deploying and deleting deployments each run. Default value: `false`
 * `CONTAINER_CONFIG_PATH`: directory where `config.json` file is located. This file contains the pull secrets to be used by
 the container engine. Default value: `$HOME/.docker/config.json`
-* `STRIMZI_NAMESPACE`: namespace where strimzi is installed. It is useful for pipelines
-where strimzi is installed before the STs. Default value: `kafka`
+* `STRIMZI_INSTALLED`: skip strimzi installation if it is already installed. Default value: `false`
```

Comment:
I think the semantics would be clearer if you called these SKIP_STRIMZI_INSTALL etc